CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+# [1.15.0 (August 31st, 2022)](https://github.com/eclipse/eclipse.jdt.ls/milestone/104?closed=1)
+ * enhancement - Search more folders to infer source roots for invisible projects. See [#2176](https://github.com/eclipse/eclipse.jdt.ls/pull/2176).
+ * enhancement - Support for Android projects. See [#923](https://github.com/eclipse/eclipse.jdt.ls/issues/923).
```

Comment:
Experimental support for Android projects. Also mention there is a setting `java.jdt.ls.androidSupport.enabled` for it, and it's automatically enabled in VS Code Insiders.  See https://github.com/redhat-developer/vscode-java/pull/2644